src/constants.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path')
+
+const PORT = 8080
+
+const COMPONENT = process.argv[2]
```

Comment:
Maybe not in this PR, but we will need to check command line inputs (and have a command line help).
Maybe using this lib or an other : https://github.com/tj/commander.js